assets/webpack.js

```diff
@@ -1,4 +1,4 @@
-const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
```suggestion
const TerserPlugin = require('terser-webpack-plugin')
```
@davydog187 :nail_care: Seems that we are not using semicolons in the rest of the config file.